include/seqan3/io/sam_file/format_bam.hpp

```diff
@@ -27,6 +27,7 @@
 #include <seqan3/core/range/type_traits.hpp>
 #include <seqan3/io/detail/ignore_output_iterator.hpp>
 #include <seqan3/io/detail/misc.hpp>
+#include <seqan3/io/detail/take_until_view.hpp>
```

Comment:
Maybe unused?

Reply:
looks like it, yes. I removed it, lets see what the CI says to it :-)